### DISCUSSION TITLE
What does engagement (in coliving and community) mean to you?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#979] [General]
This question came out of an initial sharing in the Conscious Coliving whatsapp chat. And I invite others to add their thoughts and perspectives below. 

If you would like to see the context or others responses keep reading below, otherwise jump into the comments with your thoughts.

### Context
I initially asked people to share what was alive for them specifically at this moment in time in relation to their experience of conscious coliving (along with introducing themselves).

Matt from Conscious Coliving.com shared: what is alive for me in regards to conscious coliving at this moment is exploring interpersonal relationships and 'engagement' - how to foster it, what it really means, how to maintain it, etc.

This prompted me to ask Matt: What is the understanding you have reached around what engagement means so far? and to reflect that we often use a lot of terms like this in communal aspects but we definitely all make assumptions of it being or meaning different things and **then to ask the question to the group of What does engagement (in coliving and community) mean to you?**

### Responses in the chat: 

**Rufus**: It obviously depends on what is expected in the community - some intentional communities may expect people to work 20h a week or even full-time whilst others have little specifically expected.

Engaged in a basic sense would be "showing up" both literally and with presence for key activities.

Engaged in a deeper sense i think is about being connected and enrolled in what the coliving community is about and up to.

These are just some first thoughts ...

**Simon**: "Engagement" seems like a word that can vary greatly in interpretation, but on the whole positive. I'd say the ambivalence comes from the different expectations that people might have about what playing a recognised part in a community could involve. If you're talking about a traditionalist community, engagement might only be seen in terms of playing a pre-ordained role, and to be disruptive or questioning might be regarded as anti-social, and therefore not engaged. I'm coming round to the nuance: engagement on whose terms? Am I engaged with a community if I see something going badly wrong and I'm trying my best to do what I can to set it right? The danger being that engagement on one's own terms runs the risk of being seen as disengagement in terms of the majority, or in the view of those holding influence or power. To me, engagement in life is more vital and primary than engagement with a particular community; and if there is a dissonance then it's time to find another community where there is more alignment … where what appears as deeply right to oneself in terms of engagement is also appreciated and seen and valued as engagement by the community. One thing is fairly clear to me though, and probably not at all controversial: if one is not communicating or interacting with other people in a co-living community, that is not likely to be engagement in anyone's terms. I can imagine an edge case where someone is doing solo tasks that are valued by the rest of the community; but if there isn't ongoing communication then there is always the risk that what has been appreciated in the past becomes no longer so, and thus clearly disengaged. Pause. Another way of looking at engagement would be through the lens of the Peckham Experiment definition of health: "mutual synthesis of organism and environment". If it's healthy in this way then it is engaged, for sure.

**Catherine**: "Engagement" isn't a word that sparks a lot for me in this context atm, but the conversation reminds me of a short Moving Debate that Valerie and I designed and facilitated together at the Open Residency at the Bergerac Hub a couple of weeks ago. The moving debate explored perspectives on roles and contribution in community and what is the purpose of a community collective care system. 

We were motivated to hold this exploration after some tensions surfaced in a weekly community planning meeting around what was "fair" in terms of roles and contributions to our collective care system and what it meant to take on certain roles within it. 

Something I found interesting from the moving debate was the group's reflections on the statement "The main purpose of a community collective care system is as a container or spark for learning and growth". 

While the purpose of the current collective care system in the Bergerac Hub (a system of roles for food shopping, cooking, cleaning, and house maintenance) might seem superficially to be simply about caring for survival and basic human needs, the group seemed to coalesce around the idea that the collective system's capacity to learn, grow and evolve over time was essential for resilience, and therefore also survival.

**Valerie**: I perceive engagement as a verb rather than a noun. It is within the dynamic movement of engaging oneself in a direction that we touch the liminal space of 'tranself' that can adjust to what a system needs.

Engagement is one of the core principles of the Praxis ecology we have at the hub at Bergerac, alongside the 5 Minimum Viable Practices for community. The main requirement we have is that when people come, they engage in a weekly and daily structure.

Now, the real question for me is how to maneuver between our ideal to be a community and our deep resistance to it (authority trauma, individual freedom of choice, potential illusion to trust what we think is good for us, difficulty to engage). How do we build containers (DDS) that activate this communal 'tranself' that empowers individuals in a way that serves the whole, yet protects the necessary process for retaliation and resistance? 🙏

**M**: How about engagement in terms of joining in a discussion/dialogue.  With most other things - engagement can be mimicked by compliance.  When it comes to a discourse - it seems that people are engaged purely by intentionally participating.

**Comment by martinprogrammer** [comment]
@asimong 
"Am I engaged with a community if I see something going badly wrong and I'm trying my best to do what I can to set it right? The danger being that engagement on one's own terms runs the risk of being seen as disengagement in terms of the majority, or in the view of those holding influence or power. "

I think this is very good.  Can we discuss it further?  What's the answer?  
Is the engagement best served in the spirit of the common goals or in the spirit of supporting the leaders?
What are practical ways of raising this issue and agreeing on the terms?
